content/strategy-goals/strategy/cloud/security/index.md

```diff
@@ -4,6 +4,8 @@
 
 ### 1. Secure Sourcegraph Cloud for private code (due end of 2021-July)
 
+<!-- TODO: Can we replace the references to Cloud Saas with Cloud Org?-->
+
```

Comment:
Can we replace the references to Cloud SaaS  with Cloud Org?

Reply:
Yes - Replace with Cloud org here.